python/py-igraph/Portfile

```diff
@@ -6,13 +6,13 @@ PortGroup           python 1.0
 name                py-igraph
 python.rootname     igraph
 python.pep517       yes
-version             0.10.8
+version             0.11.2
 revision            0
 categories-append   math science
 platforms           darwin
 license             GPL-2+
 
-python.versions     37 38 39 310 311
+python.versions     38 39 310 311 312
```

Comment:
as the CI tells you there are many dependencies missing for PY312 so you cannot just add this new subport

Reply:
@reneeotten These are test dependencies in both cases. The packages are perfectly usable without them.

Reply:
Since the port is installable, and works fine, I would rather have 312 added right now than to have to come back later. 

The tests are primarily for the maintainer's (i.e. my) benefit, and they don't need to work right away. @catap, are you fine to add 312 to `texttable` even though `port test` doesn't work yet with that Python version?

Reply:
Tests disabled for py312 subport, marking as resolved.